lib/fog/ovirt/compute/v4.rb

```diff
@@ -71,11 +73,8 @@ def ovirt_attrs(obj)
           end
           # rubocop:enable Metrics/AbcSize, Metrics/CyclomaticComplexity, Metrics/PerceivedComplexity
 
-          # rubocop:disable Metrics/AbcSize, Metrics/CyclomaticComplexity
           def get_attr_value(value, opts)
             case value
-            when OvirtSDK4::TemplateVersion, Array, Hash, OvirtSDK4::List, OvirtSDK4::DataCenter
```

Comment:
This turned into a huge list while testing cloudinit.
I decided to change `value` to the default and if specific values need to turn into a string I'll add a case for them.